RecoCTPPS/TotemRPLocal/src/CTPPSDiamondRecHitProducerAlgorithm.cc

```diff
@@ -40,10 +41,14 @@ void CTPPSDiamondRecHitProducerAlgorithm::build(const CTPPSGeometry& geom,
 
     // retrieve the timing calibration part for this channel
     const int sector = detid.arm(), station = detid.station(), plane = detid.plane(), channel = detid.channel();
-    const auto& ch_params = calib_.parameters(sector, station, plane, channel);
-    // offset + time precision set to 0 if not found
-    const double ch_t_offset = calib_.timeOffset(sector, station, plane, channel);
-    const double ch_t_precis = calib_.timePrecision(sector, station, plane, channel);
+    std::vector<double> ch_params;
```

Comment:
@forthommel @perrotta @slava77 in this update the constness of the variables is lost, due to the conditional definitions depending on ```apply_calib_``` . I wonder whether it might not be recovered with a conditional definition like:
```
const double ch_t_offset = (apply_calib_) ? calib_.parameters(sector, station, plane, channel) : 0. ;
```

Reply:
Thaks @fabiocos ! Addressed in 7c8eb50 & ae7a25d.